### PR TITLE
 move config options in .ini file 

### PR DESCRIPTION
**Pull request by romangraef**
for using the config.py nothing should be changed. (i did the type
conversions)

### CODE REVIEW

**Comment by romangraef** [comment]
of course CI fails without an INI: so here it comes 
```ini
[discord]
# Discord API / Login settings
token = MzEyMjU2ODcxMTc0MTExMjMz.Dau4RA.JO6LraTHOYUJoX9H9SXl-a-W8AU

# Rules 'n' settings
max_channels_per_category = 50
max_participants = 40
max_cc_per_user = 8
season = 1

ww_prefix = !
act_prefix = -


[roles]
# List of specific roles
administrator = 0
game_master = 456884637638328332
participant = 456881451745411089
dead_participant = 457498358077063170
frozen_participant = 457498631344357387
# TODO
suspended = 0

[channels]
# List of specific channels
welcome_channel = 456880220486631424
game_log = 456880259636396034
bot_spam = 446046437114118145
story_time = 446205631159074818

[database]
# Database settings
dynamic_config = dynamic.json
general_database = general.db
database = game.db
```

**Comment by Bentechy66** [comment]
You've leaked your token there (or at least a placeholder one)

File: config.py [hunk 1]
```diff
@@ -1,33 +1,43 @@
...
-game_master = 456884637638328332
-participant = 456881451745411089
-dead_participant = 457498358077063170
-frozen_participant = 457498631344357387
-suspended = 0 # TODO
+# Database settings
+database = config['database']
+dynamic_config = database['dynamic_config']
+general_database = database['general_database']
+database = database['database']
```
Comment (Randium) [inline config.py]: We do not store such information in the database. Where are you getting this information from?

File: config.py [hunk 2]
```diff
@@ -1,33 +1,43 @@
...
 
 # Rules 'n' settings
-max_channels_per_category = 50
-max_participants = 40
-max_cc_per_user = 8
-season = "1"
+max_channels_per_category = int(discord['max_channels_per_category'])
+max_participants = int(discord['max_participants'])
+max_cc_per_user = int(discord['max_cc_per_user'])
+season = discord['season']
```
Comment (Randium) [inline config.py]: How is Discord supposed to know what I want as my maximum amount of participants?

Comment (romangraef) [reply config.py]: excuse me, did you read the code?

File: config.py (hunk 1 above)
Comment (romangraef) [reply config.py]: excuse me, did you read the code?

**Comment by Randium** [comment]
This has conflicting files, isn't accepted by the evil dr. Travis and needs more clarification on how this will be changed. This pull request will be closed as long as there aren't any changes visible.

**Pull request closed without merging**